incubator/vault/templates/prometheusrules.yaml

```
@@ -0,0 +1,69 @@
+{{- if .Values.vaultExporter.alerts.enabled -}}
```

A small suggestion. How about leaving the `PrometheusRule` resource independent of `vaultExporter`. I'm thinking that the existing `vaultExporter` configuration might go away in the future in favour of something more generic (i.e. extraContainers). Could we go with something like this for enabling and defining the `PrometheusRule` resource:

```suggestion
{{- if .Values.prometheusRule.enabled -}}
```

This loosely following what other charts are doing:

https://github.com/helm/charts/blob/e5e3be5b957b3373bd233c03431189bd9c5ce7a7/stable/envoy/values.yaml#L254